python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -250,13 +250,39 @@ keys:
           convert_types:
             - str
           description: VLAN tag to configure on sub-interface.
+          deprecation:
+            warning: true
+            remove_in_version: 6.0.0
+            new_key: encapsulation_dot1q.vlan
+        encapsulation_dot1q:
+          description: |-
+            Warning: `encapsulation_dot1q` should not be combined with `ethernet_interfaces[].type == l3dot1q/l2dot1q`.
+          type: dict
+          keys:
+            vlan:
+              type: int
+              convert_types:
+                - str
+              required: true
+              description: VLAD ID.
+            inner_vlan:
+              type: int
+              convert_types:
+                - str
+              description: Inner VLAN ID. This setting can only be applied to sub-interfaces on EOS.
         encapsulation_vlan:
           type: dict
+          description: |-
+            This setting can only be applied to sub-interfaces on EOS.
+            Warning: `encapsulation_vlan` should not be combined with `ethernet_interfaces[].type == l3dot1q/l2dot1q`.
           keys:
```

Comment:
there is new option of client ID if we 
```
wan1-dc1(config-if-Et1.1-encap-vlan)#client ?
  WORD       Client ID
  dot1ad     Dot1ad encapsulation
  dot1q      Dot1q encapsulation
  unmatched  Match packets that do not match any other client encapsulation on the port
  untagged   No encapsulation
```

Reply:
I can not see it here -
```
s1-leaf1(config-if-Et2.1-encap-vlan)#client ?
  dot1ad     Dot1ad encapsulation
  dot1q      Dot1q encapsulation
  unmatched  Match packets that do not match any other client encapsulation on the port
  untagged   No encapsulation
```

Reply:
Got it

Reply:
Had a discussion with @ClausHolbechArista , we will skip the `client id` option for now.